TotalCrossSDK/src/main/java/totalcross/ui/html/Document.java

```diff
@@ -409,21 +409,21 @@ public void controlPressed(ControlEvent e) {
 
     /** Impl. note: only works when reference resolution has been disabled */
     @Override
-    protected void foundReference(byte input[], int offset, int count) {
-      char res = NamedEntitiesDereferencer.toCode(input, offset, count);
+    protected void foundReference(char input[], int offset, int count) {
+      char res = NamedEntitiesDereferencer.toCode(Convert.charConverter.chars2bytes(input, 0, input.length), offset, count);
```

Comment:
What are these converters used for and why are thy required? (also in other places)

Reply:
I did this to not change methods that are not related to the xml parser, or in some cases methods that are too old and difficult to understand what's the reasoning behind it, so I used the converter to keep using bytes and not change the implementation to minimize the change on these legacy methods